### PR TITLE
Added service endpoints APIs

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@ravbhatnagar New swagger file.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]
Waiting for ARM feedback.

**Comment by DeepakRajendranMsft** [comment]
Ping....Why does this need ARM review again? this has already been reviewed by them. we are renaming this property. so we had to remove and replace it back. @sergey-shandar  @ravbhatnagar

**Comment by sergey-shandar** [comment]
@DeepakRajendranMsft I see a new file Microsoft.Network/2017-06-01/endpointService.json. Could you send a link to the ARM review for this file?

**Comment by DeepakRajendranMsft** [comment]
this is the PR where we dropped it https://github.com/Azure/azure-rest-api-specs/pull/1486. we wouldnt have put in there if we didnt get ARM approval :)

**Comment by sergey-shandar** [comment]
@ravbhatnagar could you confirm ARM review? I still can't see a ARM review confirmation for the new file.

**Comment by ravbhatnagar** [comment]
@sergey-shandar - looks good.

**Comment by DeepakRajendranMsft** [comment]
@sergey-shandar can this be merged please

**Comment by sergey-shandar** [comment]
@DeepakRajendranMsft I'm working on a review. I'll post it today.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetwork.json
Comment (DeepakRajendranMsft) [inline specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetwork.json]: @amarzavery  @salameer this is well documented and we are okay with the breaking change.
it is read-write property.

Comment (DeepakRajendranMsft) [inline specification/network/resource-manager/Microsoft.Network/2017-06-01/virtualNetwork.json]: @ravbhatnagar can you sign off on this new additional property.

**Comment by ravbhatnagar** [comment]
@DeepakRajendranMsft - Is this again the same case where it has already been added on the service side for this api-version and swagger update is happening now?

**Comment by DeepakRajendranMsft** [comment]
@ravbhatnagar yes. because we did a rename of the property.

**Comment by ravbhatnagar** [comment]
@sergey-shandar - ARM signs off.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**
